cacli/ibmcloud/request.go

```diff
@@ -0,0 +1,82 @@
+package ibmcloud
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+func handleRequest(request *http.Request, header map[string]string, res interface{}) error {
+	for key, value := range header {
+		request.Header.Add(key, value)
+	}
+
+	resp, err := client.Do(request)
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	if !isSuccess(resp.StatusCode) {
```

Comment:
oh you should probs move this function to this file too

Reply:
except you probably don't need the function anymore because you already made this nice big function and isSuccess is pretty much just an if statement checking the status range